speaking-portal-project/src/test/animation.factory.test.ts

```diff
@@ -35,12 +38,13 @@ describe('Animation Factory Tests', () => {
             throw error
         }
         })
+
+    // TODO: Capture and FFPEG Error Object 
     test('failing ffmpeg processor', async() => 
         await expect(() => ffmpegProcessor([''],'')).rejects.toThrowError())
 
     // TODO: Figure out a way to test getVideoExport. Maybe make function more generic rather then hard coding paths
```

Comment:
an update on this: I fixed that a while back and should be able to be tested now